openedx/features/colaraz_features/helpers.py

```diff
@@ -427,3 +431,16 @@ def get_query_dict(_dict):
         else:
             q[key] = value
     return q
+
+
+def get_role_based_urls(response):
+    email = response.get('email')
+    access_token = response.get('access_token')
+    if email and access_token and hasattr(settings, 'COLARAZ_APP_LINKS_API'):
+        api_url = '{}?{}'.format(getattr(settings, 'COLARAZ_APP_LINKS_API'), urlencode({'email': email}))
```

Comment:
nit. you can totally ignore it if you want. We have already made sure that `COLARAZ_APP_LINKS_API` is present. So I just don't get why we are using getattr.
 But again, if it's intentional, you can ignore this
```suggestion
        api_url = '{}?{}'.format(settings.COLARAZ_APP_LINKS_API, urlencode({'email': email}))
```

Reply:
updated